hkmc2/shared/src/main/scala/hkmc2/semantics/Resolver.scala

```diff
@@ -95,7 +95,7 @@ object Resolver:
     def showEnv: Str =
       iEnv.values
         .flatMap(_.map((typ, instance) => s"${typ.show}"))
-        .mkString("(", ", ", ")")
+        .mkString("[", ", ", "]")
```

Comment:
Why the insistence on brackets? It looks like an array now, but is not an array. Wouldn't it look better as:
```suggestion
        .mkString(", ")
```

Reply:
Though we should then also add a special-case for the empty list.

Reply:
Applied. Plus I made it output `<no available>` when there is no instance:

```cs
M.f
//│ ╔══[ERROR] Cannot query instance of type Int for call: 
//│ ║  l.13: 	M.f
//│ ║        	^^^
//│ ╟── Required by contextual parameter declaration: 
//│ ║  l.10: 	  fun f(using Int)
//│ ║        	              ^^^
//│ ╙── Missing instance: Expected: Int; Available: <no available>
```

Reply:
Oops but it should be  `‹none available›`, not `<no available>`!